plugins/destination/duckdb/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ghcr.io/gythialy/golang-cross:1.20.1-0
+
+# install goreleaser
+# https://github.com/goreleaser/goreleaser-pro/releases/download/v1.12.3-pro/goreleaser-pro_Linux_x86_64.tar.gz
+ARG GORELEASER_VERSION=1.12.3-pro
```

Comment:
Opened https://github.com/cloudquery/cloudquery/pull/8833 so we can auto update this version